dashboard/app/models/pd/workshop_material_order.rb

```diff
@@ -57,25 +61,42 @@ class WorkshopMaterialOrder < ActiveRecord::Base
     validates_format_of :phone_number, with: PHONE_NUMBER_VALIDATION_REGEX, if: -> {phone_number.present? && !User.with_deleted.find_by_id(user_id).try(:deleted?)}
     validates :zip_code, us_zip_code: true, if: -> {zip_code.present? && !User.with_deleted.find_by_id(user_id).try(:deleted?)}
 
-    validate :valid_address?, if: -> {address_fields_changed? && !user.try(:deleted?)}
+    validate :valid_address?, if: -> {address_fields_changed? && !address_override && !user.try(:deleted?)}
+
+    attr_writer :address_override
+
+    def address_override
+      @address_override == "1"
+    end
```

Comment:
nit: I'd prefer (ruby-style) the bool method to end in a `?`. I'd also prefer if the transient attribute was read/write, and also not to overload the name.

E.g.
```ruby
attr_accessor :address_override

def address_override?
  address_override == '1'
end
```